### ISSUE TITLE
iframes lack aria label

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#15188]
For example, on the overview page, we have: 
```html
<iframe class="container-frame" name="cockpit1:localhost/system" data-host="localhost" 
src="../../@localhost/system/index.html#/" data-active="true" style="display: block;" 
data-ready="1" data-loaded="1"></iframe>
```
(as rendered in DOM)

Note: This affects the iframe for all pages.

---

It needs one of the following:

1. title attribute
2. aria-label (labeling it inline)
3. aria-labelledby (labeling it by pointing to a textual element that describes it, using an ID)
4. ~role="none" or role="presentation"~ (not valid in this context; our iframes need to be used and are relevant)

I'd prefer # 1: Adding a title attribute (and making sure it is accurate).
